packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx

```diff
@@ -201,10 +202,20 @@ const ProofOfAddressForm = observer(
             // upload files
             try {
                 const api_response = await upload(document_files);
+
                 if (api_response?.warning) {
-                    setStatus({ msg: api_response?.message });
                     setFormState({ ...form_state, ...{ is_btn_loading: false } });
-                    setShouldScrollToTop(true);
+
+                    if (api_response.warning === API_ERROR_CODES.DUPLICATE_DOCUMENT) {
+                        if (is_for_cfd_modal && typeof step_index !== 'undefined') {
```

Comment:
```suggestion
                        if (is_for_cfd_modal && typeof step_index !== 'undefined') {
```
question: why would you expect `step_index` to be undefined in this case? The typings of this component `step_index` is defined as a number. Can we prevent `step_index` from being undefined?

Reply:
@yashim-deriv on checking, the type of step_index is string|undefined in the component.
<img width="598" alt="Screenshot 2024-06-13 at 10 23 16 AM" src="https://github.com/binary-com/deriv-app/assets/84661147/c865df48-6f80-4684-ac63-69c36196ab4e">

Reply:
Yes - in terms of logic,  can we prevent step_index from being undefined? TypeScript is added to provide some rules to the components, that is why i asked:
- Why do we need step_index to be undefined? Is there any reason code-wise? Steps for a component sounds like it should be always a number, because there is a component paired to the `step_index`

Reply:
we are using same POA component in MT5 account creation modal  and account settings section. `step-index` could be undefined if its in account settings section.
On checking we already have a flag called `is_for_cfd_modal`
I have updated the code to use `is_for_cfd_modal` and moved the index logic to parent -modal wrapper